package.json

```diff
@@ -1,54 +1,43 @@
 {
-  "name": "nereo-material",
+  "name": "cotecna-material",
   "version": "0.0.0",
```

Comment:
Could we set an initially version "1.0.0-beta.1" ??

Reply:
Done 
![image](https://user-images.githubusercontent.com/15807748/144236563-65b9b3cf-dbc6-4018-883f-4775a075c6de.png)

Reply:
Perfect, thanks, I will close it